VladKornieiev/src/ACO/week2/homework/library/model/Library.java

```diff
@@ -0,0 +1,81 @@
+package ACO.week2.homework.library.model;
+
+import ACO.data_structures.MyArrayList;
+import ACO.week2.homework.library.model.printed_editions.PrintedEdition;
+
+/**
+ * Created by v21k on 31.01.17.
+ */
+public class Library {
+    private String name;
+    private String city;
+    private MyArrayList readers = new MyArrayList();
+    private MyArrayList blackList = new MyArrayList();
```

Comment:
Better to create field in Reader class (boolean inBlacklist), so it'll be more easy and more flexiable